### PR TITLE
Translation improvement

### PR DESCRIPTION
**Pull request by victor-paumier**
PR-151 #add admin text to translation
add admin translation domain
improve import file
improve translate website pages design

### CODE REVIEW

File: src/Controller/SectionController.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function index(SectionRepository $sectionRepo): Response
      *
      * @return RedirectResponse|Response
      */
-    public function new(ObjectManager $om, Request $request)
+    public function new(ObjectManager $om, TranslatorInterface $translator, Request $request)
```
Comment (JeremieSamson) [inline src/Controller/SectionController.php]: multiline plz

File: src/Form/Handler/TransUnitFormHandler.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+class TransUnitFormHandler implements FormHandlerInterface
+{
+    protected $transUnitManager;
+    protected $fileManager;
+    protected $storage;
+    protected $localeManager;
+    protected $rootDir;
+
+    public function __construct(TransUnitManagerInterface $transUnitManager, FileManagerInterface $fileManager, StorageInterface $storage, LocaleManager $localeManager, $rootDir)
```
Comment (JeremieSamson) [inline src/Form/Handler/TransUnitFormHandler.php]: multiline plz

File: src/Controller/SectionController.php (hunk 1 above)
Comment (victor-paumier) [reply src/Controller/SectionController.php]: done

File: src/Form/Handler/TransUnitFormHandler.php (hunk 2 above)
Comment (victor-paumier) [reply src/Form/Handler/TransUnitFormHandler.php]: done

**Pull request merged**